include/nntile/kernel/gemm.hh.in

```diff
@@ -100,6 +100,20 @@ void cublas(cublasHandle_t handle, cublasOperation_t transA,
             CUBLAS_GEMM_DEFAULT_TENSOR_OP);
 }
 
+// Overloaded call to cuBLAS GEMM for bf16_t
+static inline
+void cublas(cublasHandle_t handle, cublasOperation_t transA,
+        cublasOperation_t transB, int M, int N, int K, float alpha,
+        const bf16_t *A, int ldA, const bf16_t *B, int ldB,
+        float beta, bf16_t *C, int ldC)
+    noexcept
+{
+    cublasGemmEx(handle, transA, transB, M, N, K, &alpha, (const float *)A,
```

Comment:
pointers A, B and C are not compatible with float* type. Pointers A, B and C are BF16*. Just remove these cast.

Reply:
Or cast them into __nv_bfloat16*.